src/EnergyPlus/OutdoorAirUnit.cc

```diff
@@ -742,7 +749,13 @@ namespace OutdoorAirUnit {
                                     GetCoilSteamInletNode(OutAirUnit(OAUnitNum).OAEquip(CompNum).ComponentIndex,
                                                           OutAirUnit(OAUnitNum).OAEquip(CompNum).ComponentName,
                                                           ErrorsFound);
+                                OutAirUnit(OAUnitNum).OAEquip(CompNum).CoilWaterOutletNode =
```

Comment:
:+1: So this addresses the outlet node initialization crash.

Reply:
So this was never tested, and a unit test would have failed.

Reply:
How can an example file ever pass with this? Because we don't have one.

Reply:
@Nigusse point me to an example file that has this configuration and tests this defect? I know it doesn't exist, I just want to start the conversation.

Reply:
Just added a defect file that created. [Here](https://github.com/NREL/EnergyPlusDevSupport/tree/master/DefectFiles/6000s/6843).